content/guides/tagging.md

```diff
@@ -136,7 +146,7 @@ To filter the list of hosts in the Infrastructure list, enter a tag in the filte
 
 ### Using tags in Monitors
 
-When defining a monitor, you can use tags to allow the monitor to apply to any subset of hosts across your environment.
+When defining a monitor, you can use tags in the ```from:``` textbox.  to allow the monitor to apply to any subset of hosts across your environment.
```

Comment:
> to allow the monitor to apply to any subset of hosts across your environment.

"to limit the monitor's scope."

(it's not just for limiting hosts; not all metrics are per-host)